### PR TITLE
today I received nil instead of message

### PR DESCRIPTION
**Pull request by apsheronets**

### CODE REVIEW

**Comment by ivanovaleksey** [comment]
Can you provide a way to reproduce the issue?

**Comment by apsheronets** [comment]
Well, I'm afraid I'm not. How could I know what kind of unsupported update I received from Pasha Durov?
And it doesn't really matter. If we get unsupported type of update, `message` becomes `nil` and logging raise `undefined method `from' for nil:NilClass`.

**Comment by ivanovaleksey** [comment]
You are right.
But I just wonder. It would be nice to debug it to see what causes the error.

File: lib/telegram/bot/client.rb [hunk 1]
```diff
@@ -58,10 +58,10 @@ def extract_message(update)
       end
 
       def log_incoming_message(message)
-        uid = message.from ? message.from.id : nil
-        logger.info(
+        logger.info {
+          uid = message.try(:from) ? message.from.id : nil
```
Comment (ivanovaleksey) [inline lib/telegram/bot/client.rb]: `try` is active-support method. The gem doesn't use active-support so this `try` won't work and produces an error.

That is why it was previously `message.from ? message.from.id : nil` and not `message.from.try(:id)`.

Comment (apsheronets) [inline lib/telegram/bot/client.rb]: Oh wow. Sorry.

**Comment by apsheronets** [comment]
As a suggestion: channel_post.
https://core.telegram.org/bots/api/#update

**Comment by ivanovaleksey** [comment]
Very likely. There is #84 where `channel_post` are going to be added to `Types::Update`.

**Pull request closed without merging**